brands/shop/books.json

```diff
@@ -121,6 +121,20 @@
       "shop": "books"
     }
   },
+  "shop/books|Gramedia": {
+    "countryCodes": ["id"],
+    "tags": {
+      "brand": "Gramedia",
+      "brand:en": "Gramedia",
+      "brand:id": "Gramedia (toko buku)",
+      "brand:wikidata": "Q19646130",
+      "brand:wikipedia": "id:Gramedia (toko buku)",
+      "name": "Gramedia",
+      "name:en": "Gramedia",
+      "name:id": "Gramedia (toko buku)",
```

Comment:
Does it really need `brand:en`, `brand:id`, `name:en` and `name:id` tags?   I don't understand them..
It looks like this brand is just "Gramedia" always?

Reply:
ah my bad, i will fix it